api-reference/direct-connects/hotel-service-2/Authentication-and-Errors.markdown

```diff
@@ -3,34 +3,97 @@ title: Authentication and Errors
 layout: reference
 ---
 
+**rename page to Error handling**
```

Comment:
This should be removed and page should be renamed. You'll want to do the following:

1. Add a title to the markdown file. E.g.

```yml
---
layout: reference
title: Error Handling
---
```

2. Update the title/url in the sidebar here: https://github.com/concur/developer.concur.com/blob/preview/_data/sidebars/api-reference.yml#L113

Reply:
Done. Thanks for picking that up.